### PR TITLE
Get-DbaResourceUsage Ms

### PR DESCRIPTION
**Pull request by potatoqualitee**
Change Microseconds to milliseconds

### CODE REVIEW

**Comment by potatoqualitee** [comment]
image

**Comment by potatoqualitee** [comment]
test failed cuz appveyor

**Comment by potatoqualitee** [comment]
Waiting for feedback from Michael J. Swart himself

File: functions/Get-DbaTopResourceUsage.ps1 [hunk 1]
```diff
@@ -98,10 +98,10 @@
 						    coalesce(db_name(st.dbid), db_name(cast(pa.value AS INT)), 'Resource') AS [Database],
 						    coalesce(object_name(ST.objectid, ST.dbid), '<none>') as ObjectName,
 						    qs.query_hash as QueryHash,
-						    qs.total_elapsed_time as TotalElapsedTime,
+						    qs.total_elapsed_time / 1000 as TotalElapsedTimeMs,
```
Comment (mjswart) [inline functions/Get-DbaTopResourceUsage.ps1]: Right, if ms is short for milliseconds, then elapsed time (in microseconds) / 1000 is indeed in milliseconds.

**Comment by potatoqualitee** [comment]
awesome thank you 👍

**Pull request merged**